### DISCUSSION TITLE
No space on device when loading hello-world on P3265

### DISCUSSION THREAD

**Discussion by egSat** [AxisCommunications/acap-computer-vision-sdk-examples#40] [Q&A] [answered]
Hi there,
I'm trying to install this computer vision example, https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/master/hello-world
But I fail to install it, I get the error "no space on device". I attach an ssh session screenshot to the camera, showing the available space before and after trying to install the example.
Camera firmware is 10.10.73. Docker daemon 1.2.3
Attached screenshot before and after trying to install the example, and dockerlog from the camera

Thank you very much
![thumbnail_image](https://user-images.githubusercontent.com/107174547/172789005-6d004cbd-f2e0-4266-a2e5-87925836d83d.png)

**Comment by Corallo** [comment, marked as answer]
Hello @egSat 

The space in the camera for applications is limited and containers are often hundreds of MB.
P3265 support microSD expansions, we recommend to use it and set the docker acap to use store the containers on the SD storage. 
You can do it on the web gui -> systems -> plain config -> search -> "docker" -> SD card support -> Yes.
You might also need to format the SD card to ext4 if it is not yet that format.
That can be done on system -> storage.

**Comment by Corallo** [comment]
I am converting this issue into a Q&A discussion

**Comment by egSat** [comment]
Hi @Corallo , thank you very much for the answer, I'll try to install it on the sd storage.
